### PR TITLE
added locale to home page title

### LINKED ISSUE #611: In some navigation dialogues in Admin area, there is no way to see which homepage is for which language
For example:

image

### PR DESCRIPTION
**Pull request by mbilalhameed**
fixes #611

### CODE REVIEW

**Comment by mbilalhameed** [comment]
@sheralim012

File: iogt_content_migration/management/commands/load_v1_db.py [hunk 1]
```diff
@@ -1250,6 +1250,14 @@ def translate_home_pages(self, home):
         for row in cur:
             locale, __ = Locale.objects.get_or_create(language_code=self._get_iso_locale(row['locale']))
             self.translate_page(locale=locale, page=home)
+            self.add_home_page_title_and_locale(home, locale)
+
+    def add_home_page_title_and_locale(self, home_page, locale):
+        translated_home_page = home_page.get_translation_or_none(locale)
+        if translated_home_page:
+            translated_home_page.title = home_page.title + ' [' + locale.__str__() + ']'
```
Comment (sheralim012) [inline iogt_content_migration/management/commands/load_v1_db.py:1258]: use f-string

Comment (sheralim012) [reply iogt_content_migration/management/commands/load_v1_db.py:1258]: update draft title too

File: iogt_content_migration/management/commands/load_v1_db.py [hunk 2]
```diff
@@ -1250,6 +1250,14 @@ def translate_home_pages(self, home):
         for row in cur:
             locale, __ = Locale.objects.get_or_create(language_code=self._get_iso_locale(row['locale']))
             self.translate_page(locale=locale, page=home)
+            self.add_home_page_title_and_locale(home, locale)
+
+    def add_home_page_title_and_locale(self, home_page, locale):
```
Comment (sheralim012) [inline iogt_content_migration/management/commands/load_v1_db.py:1255]: simplify this no need to create method

**Review by sheralim012 — CHANGES_REQUESTED** [review]

File: iogt_content_migration/management/commands/load_v1_db.py [hunk 3]
```diff
@@ -1250,6 +1250,14 @@ def translate_home_pages(self, home):
         for row in cur:
             locale, __ = Locale.objects.get_or_create(language_code=self._get_iso_locale(row['locale']))
             self.translate_page(locale=locale, page=home)
+            translated_home_page = home.get_translation_or_none(locale)
+
+            if translated_home_page:
+                modified_title = f"{home.title} [{locale.__str__()}]"
```
Comment (sheralim012) [inline iogt_content_migration/management/commands/load_v1_db.py:1256]: use `translated_home_page.title` instead of `home.title`

**Review by sheralim012 — CHANGES_REQUESTED** [review]

**Review by sheralim012 — APPROVED** [review]

**Pull request merged**